docs/interplanetary-file-system.md

```diff
@@ -0,0 +1,67 @@
+---
+id: interplanetary-file-system
+title: IPFS (Interplanetary File System)
+sidebar_label: IPFS (Interplanetary File System)
```

Comment:
have you checked if this sidebar label fits in the UI?

Reply:
Attended.